todo-mvc-kitchensink/tests/functional/Page.ts

```diff
@@ -76,7 +74,7 @@ export default class Page {
 
 	init(): Promise<any> {
 		return this.remote
-			.get(require.toUrl('../../src/index.html'))
+			.get('http://localhost:9000/_build/src/index.html')
```

Comment:
This should be temporary until changes are made for intern, I guess we could read this from the intern configuration... I really only mirrored what we did for the todo-mvc app.